### ISSUE TITLE
transmission fails to build

### ISSUE THREAD

**Issue by RSully** [Homebrew/homebrew#46020]
Found this: https://lists.freebsd.org/pipermail/freebsd-ports/2015-October/100790.html

```
> brew reinstall transmission
==> Reinstalling transmission
==> Downloading https://transmission.cachefly.net/transmission-2.84.tar.xz
Already downloaded: /Library/Caches/Homebrew/transmission-2.84.tar.xz
==> ./configure --prefix=/path/to/homebrew/Cellar/transmissio
==> make
Last 15 lines from /Library/Logs/Homebrew/transmission/02.make:
  CC       variant.o
upnp.c:91:51: error: too few arguments to function call, expected 7, have 6
  ret = upnpDiscover (msec, NULL, NULL, 0, 0, &err);
        ~~~~~~~~~~~~                              ^
/path/to/homebrew/include/miniupnpc/miniupnpc.h:61:1: note: 'upnpDiscover' declared here
MINIUPNP_LIBSPEC struct UPNPDev *
^
/path/to/homebrew/include/miniupnpc/miniupnpc_declspec.h:14:28: note: expanded from macro 'MINIUPNP_LIBSPEC'
                #define MINIUPNP_LIBSPEC __attribute__ ((visibility ("default")))
                                         ^
  CC       variant-benc.o
1 error generated.
make[1]: *** [upnp.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make: *** [all-recursive] Error 1
```

**Comment by dunn** [comment]
Can you post `brew gist-logs transmission`, please? Thanks!

**Comment by RSully** [comment]
I can't at the moment, sorry. See this transmission ticket: https://trac.transmissionbt.com/ticket/6003 and patch here: https://trac.transmissionbt.com/changeset/14565

**Comment by dunn** [comment]
If you want to open a PR to backport that change to the stable version, that'd be great!

**Comment by RSully** [comment]
Heads up, Transmission 2.90 is released now.

**Comment by dunn** [comment]
Feel like opening a PR for that? :heart: Take a look at https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md and let us know if you have questions!

**Comment by RSully** [comment]
Fixed in #49672.
